pom.xml

```diff
@@ -13,33 +13,27 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.3.2.RELEASE</version>
+		<version>2.3.3.RELEASE</version>
 	</parent>
 
 	<properties>
 
 		<!-- Core -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<java.version>11</java.version>
-		<maven-plugin-compiler.version>3.8.1</maven-plugin-compiler.version>
```

Comment:
These properties are ineffective; plugin and dependency versions are dictated by the Spring Boot version.